pilot/docker/Dockerfile.proxyv2

```diff
@@ -26,12 +27,13 @@ COPY --from=default /usr/lib/x86_64-linux-gnu/xtables/ /usr/lib/x86_64-linux-gnu
 COPY --from=default /usr/lib/x86_64-linux-gnu/ /usr/lib/x86_64-linux-gnu
 COPY --from=default /etc/iproute2 /etc/iproute2
 
-COPY istio-iptables /usr/local/bin/istio-iptables
-
 # Copy Envoy bootstrap templates used by pilot-agent
-COPY envoy_bootstrap_v2.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
-COPY envoy_bootstrap_drain.json /var/lib/istio/envoy/envoy_bootstrap_drain.json
-COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
+COPY --from=default /var/lib/istio/envoy/envoy_bootstrap_tmpl.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
```

Comment:
These files had permissions that made them unreadable on the distroless container. The distroless container also cannot `RUN chmod` like the default container can, as it is missing the binaries. Since the files are identical to those in the default container, I just changed them to copy over wholesale.

Reply:
I am not understanding why you had to change any of this. How did it work before such that it no longer works with your change? you are not really changing much, just copying the iptables binary with a different name, right?

Reply:
I've attempted to locally run the the commands that prow runs in `e2e-simpleTests-distroless_istio_release-1.4` on the upstream `release-1.4` branch **without** this PR, and the test still fails, so I don't know how it was working before at all. Other PRs to `release-1.4` aren't hitting this problem.

Does Prow rebuild the proxyv2 docker image even if nothing changes on them? I noticed that `distroless:cc` had an update on Jan 22, maybe the working builds are based on cached images built before that date?

Reply:
or rebuilds it every time, so in not sure how other PRs are passing

Reply:
seems like all the tests pass in this pr?

Reply:
With this change, yes. If I back it out (so the files have the standard permissions), it fails.